dashboard/src/types/tree/TreeDetails.tsx

```diff
@@ -96,18 +96,16 @@ export type TTreeTestsFullData = {
 
 export type PossibleTabs = Extract<
   MessagesKey,
-  'treeDetails.builds' | 'treeDetails.boots' | 'treeDetails.tests'
+  'global.builds' | 'global.boots' | 'global.tests'
 >;
 
 const possibleTabs = [
-  'treeDetails.builds',
-  'treeDetails.boots',
-  'treeDetails.tests',
+  'global.builds',
+  'global.boots',
+  'global.tests',
```

Comment:
this is being used as a value and not as an i18key so we can just use like this:
```suggestion
  'builds',
  'boots',
  'tests',
```

Reply:
In this case the PossibleTabs type is an Extract of MessagesKey, so this variable needs to be an intl key unless the typing is also changed